package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stylelint-config-wildbit",
-  "version": "1.2.0",
+  "version": "8.1.1",
```

Comment:
Should this reflect the version of our module, or mimic the `stylelint` version?

Reply:
That's a big bump! :)

Reply:
I fugured that it’s going to be much easier to use correct version of config if we match Stylelint version. Maybe at least for the first two digits of semver. Any objections?

Reply:
Okay, that works for me. :)

Reply:
Sounds good!